src/jit/compiler.h

```diff
@@ -1176,6 +1176,10 @@ struct FuncInfoDsc
     BYTE                unwindCodes[offsetof(UNWIND_INFO, UnwindCode) + (0xFF*sizeof(UNWIND_CODE))];
     unsigned            unwindCodeSlot;
 
+#define MAX_CFICODE_INDEX   0x200
+    BYTE                cfiCodes[(MAX_CFICODE_INDEX * sizeof(CFI_CODE))];
+    unsigned            cfiCodeIndex;
+
```

Comment:
Shouldn't this be under some kind of Unix #ifdef?

You've got an unnecessary set of extra parens.

Why is `MAX_CFICODE_INDEX` defined as that number?